src/hera.cpp

```diff
@@ -29,6 +29,18 @@
 #include <wasm-printing.h>
 #include <wasm-validator.h>
 
+#include <src/binary-writer.h>
+#include <src/common.h>
+#include <src/error-handler.h>
+#include <src/feature.h>
+#include <src/filenames.h>
+#include <src/ir.h>
+#include <src/option-parser.h>
+#include <src/resolve-names.h>
+#include <src/stream.h>
+#include <src/validator.h>
+#include <src/wast-parser.h>
```

Comment:
Any way to use a better path like `wabt/binary-writer.h` ?

Reply:
Not without moving the files manually.

Reply:
And with patching upstream cmake?